jerry-core/ecma/builtin-objects/ecma-builtin-aggregateerror.c

```diff
@@ -33,6 +33,11 @@
 #define ECMA_BUILTINS_INTERNAL
 #include "ecma-builtins-internal.h"
 
+/**
+ * This object has a custom dispatch function.
+ */
+#define BUILTIN_CUSTOM_DISPATCH
```

Comment:
If this is enabled everywhere, this should be removed, and the corresponding macros as well in a follow-up patch.